.github/pull_request_template.md

```diff
@@ -1,12 +0,0 @@
-## Ticket
```

Comment:
🙏 if newb question but does the pull request template still work after deleting this? 🤔